includes/admin/helper/class-wc-helper-updater.php

```diff
@@ -57,6 +57,10 @@ public static function transient_update_plugins( $transient ) {
 				'upgrade_notice' => $data['upgrade_notice'],
 			);
 
+			if ( $data['requires_php'] ) {
```

Comment:
Is there any chance that `$data['requires_php']` is not set and thus we should check if it is set to avoid a undefined index notice?

Reply:
Hey @rodrigoprimo! Thank you for your review!

Good catch! It's really possible! I updated it here: https://github.com/woocommerce/woocommerce/pull/28151/commits/142f810206edff672331883a60e83592bc0d79f4